### PR TITLE
[INF][INTL.INF][SETUPLIB] We don't have preinstalled IMEs

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose
Do not write the IME keyboard layouts in `INTL.CPL` because we don't have the preinstalled IMEs.
JIRA issue: [CORE-18506](https://jira.reactos.org/browse/CORE-18506)

## Proposed changes

- Don't write the preinstalled IME keyboard layouts in Chinese, Japanese and Korean (CJK).

## Comparison

BEFORE:
before

AFTER:
after
FIXED.

### CODE REVIEW

**Comment by katahiromz** [comment]
Cc @reactos/lang-chinese 
Could you test this PR?

**Comment by HBelusca** [comment]
Suppose we had those preinstalled IMEs. Would these values shown in the screenshot be correct then?

**Comment by katahiromz** [comment]
> Would these values

The preinstalled layout value had been shown in `Substitutes`.

**Comment by katahiromz** [comment]
Chinese PRC BEFORE:
prc-before
Chinese PRC AFTER:
prc-after

We are setting wrong values in another place.

**Comment by katahiromz** [comment]
Chinese PRC AFTER:


Chinese PRC AFTER:

Now fixed. Okay.

File: base/setup/lib/lang/ja-JP.h [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 MUI_LAYOUTS jaJPLayouts[] =
 {
-//    { L"0411", L"e0010411" },
+//    { L"0411", L"e0010411" }, // We don't have preinstalled IMEs.
```
Comment (HBelusca) [inline base/setup/lib/lang/ja-JP.h:5]:
```suggestion
//    { L"0411", L"E0010411" }, // We don't have preinstalled IMEs.
```
(to make it similar with the rest)

**Review by HBelusca — APPROVED** [review]
LGTM from me.

**Comment by kernelbin** [comment]
Sure, I will test it. perhaps tomorrow?

**Pull request merged**